src/screens/ChannelListScreenV2/IncognitoEmptyChat/index.tsx

```diff
@@ -10,6 +10,7 @@ import {fonts, normalizeFontSize} from '../../../utils/fonts';
 import {ANONYMOUS} from '../../../hooks/core/constant';
 import useAnonymousChannelListScreenHook from '../../../hooks/screen/useAnonymousChannelListHook';
 import {DISCOVERY_TAB_TOPICS, NavigationConstants} from '../../../utils/constants';
+import StorageUtils from '../../../utils/storage';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `StorageUtils` is a good addition to handle the storage-related operations. However, it's important to ensure that the `StorageUtils` module has been properly tested and is reliable for use in this context.
```diff
+ import StorageUtils from '../../../utils/storage';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->